### PR TITLE
New package: python3-pytest-flake8-1.0.7

### PR DESCRIPTION
**Pull request by paper42**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by FollieHiyuki** [comment]
Why not use the default `do_check()` from build style? It doesn't work?

**Comment by paper42** [comment]
@FollieHiyuki you were right, I forgot. Now I added PYTHONPATH, so I need it.

File: srcpkgs/python3-pytest-flake8/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+maintainer="Paper <paper@tilde.institute>"
+license="BSD-3-Clause"
+homepage="https://pypi.org/project/pytest-flake8/"
+changelog="https://raw.githubusercontent.com/tholo/pytest-flake8/master/CHANGELOG"
+distfiles="${PYPI_SITE}/p/pytest-flake8/pytest-flake8-${version}.tar.gz"
+checksum=f0259761a903563f33d6f099914afef339c085085e643bee8343eb323b32dd6b
+
+do_check() {
+	PYTHONPATH=. pytest
+}
```
Comment (ahesford) [inline srcpkgs/python3-pytest-flake8/template:20]: You don't need this. The build style will automatically call `python3 -m pytest` when `python3-pytest` is available, which implicitly adds `.` to `PYTHONPATH`.

**Review by ahesford — CHANGES_REQUESTED** [review]
If this package isn't being pulled into `checkdepends` for other end-user packages, does it provide value on its own?

**Comment by paper42** [comment]
> If this package isn't being pulled into `checkdepends` for other end-user packages, does it provide value on its own?

no, but the tests depending on this package are patched out from at least python3-numpy-stl. Then, at least python3-pep517, python3-cairocffi and radicale have their tests disabled **partially** because of this missing package.

**Comment by ahesford** [comment]
If adding this package is sufficient to make tests in `python3-numpy-stl` work without the `pre_check` patch, please drop that patch in this PR. No need to revbump.

**Comment by paper42** [comment]
> If adding this package is sufficient to make tests in `python3-numpy-stl` work without the `pre_check` patch, please drop that patch in this PR. No need to revbump.

@ahesford done

**Pull request merged**